src/features/search/helpers/useMaxPrice/useMaxPrice.test.ts

```diff
@@ -1,23 +1,24 @@
-import { UseQueryResult } from 'react-query'
 import { mocked } from 'ts-jest/utils'
 
-import { UserProfileResponse } from 'api/gen'
-import { useUserProfileInfo } from 'features/profile/api'
+import { useAuthContext } from 'features/auth/AuthContext'
 import { isUserExBeneficiary } from 'features/profile/utils'
 import { useMaxPrice } from 'features/search/helpers/useMaxPrice/useMaxPrice'
+import { beneficiaryUser, nonBeneficiaryUser } from 'fixtures/user'
 import { renderHook } from 'tests/utils'
 
-jest.mock('features/profile/api')
 jest.mock('features/profile/utils')
-const mockedUseUserProfileInfo = mocked(useUserProfileInfo)
 const mockedIsUserExBeneificiary = mocked(isUserExBeneficiary)
 
+jest.mock('features/auth/AuthContext')
+const mockUseAuthContext = useAuthContext as jest.MockedFunction<typeof useAuthContext>
+
 describe('useMaxPrice when no user', () => {
   beforeAll(() => {
-    mockedUseUserProfileInfo.mockReturnValue({ data: undefined } as UseQueryResult<
-      UserProfileResponse,
-      unknown
-    >)
+    mockUseAuthContext.mockReturnValueOnce({
```

Comment:
Un Once dans un beforeAll, c'est suspect : je pense qu'on peut directement mettre le mock dans le seul test de ce describe 
(garder Once dans un beforeAll va poser problème si qqun ajoute un test à ce describe)

Reply:
N.-B. : retour pour les 3 itérations dans ce fichier